package.json

```diff
@@ -13,7 +15,10 @@
     "pr": "yarn typecheck && yarn lint && yarn test && yarn checkdeps && yarn checklangs && yarn build && cd client && yarn lint && yarn build",
     "watch": "tsc-watch --pretty --onSuccess \"yarn lint\"",
     "watchfix": "tsc-watch --pretty --onSuccess \"yarn lintfix\"",
-    "postinstall": "cd client && yarn"
+    "postinstall": "cd client && yarn",
+    "i18n:pull": "tx pull --all --parallel --no-interactive --force --minimum-perc 95",
```

Comment:
Instead of setting the minimum percentage here, why not set it in the .tx/config file (where it's currently set to `0`)?

Reply:
It was just easier for testing to not set it in the .tx/config file. I change is now.